src/components/ToggleSwitch.js

```diff
@@ -1,85 +1,62 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
 
-export default function ToggleSwitch({ toggled, onChange }) {
-  return <Switch id="test-switch" toggled={toggled} onChange={onChange} />
+ToggleSwitch.propTypes = {
+  selected: PropTypes.bool.isRequired,
```

Comment:
I would change the name to "isSelected"  to make it a little bit easier to read.